src/test/groovy/com/jvm_bloggers/core/data_fetching/blog_posts/NewBlogPostStoringActorSpec.groovy

```diff
@@ -100,14 +102,39 @@ class NewBlogPostStoringActorSpec extends Specification {
         1 * blogPostRepository.save(blogPost)
     }
 
+    def "Should update only description if post already exists with different protocol"() {
+        given:
```

Comment:
This is hard to configure for the newcomers and generally I would like to make this process as easy as possible. And of course we can start religuous discussion about it :)